src/pages/AppProfile.vue

```diff
@@ -225,6 +228,8 @@ export default {
     const organisationDetails = ref({
       name: " ",
       size: 0,
+      // Nothing else can really be invalid, but this needs to be a systematic thing
```

Comment:
Personally, I prefer not to use comments for notes unless they help with reading the code. This could go on a JIRA ticket or our wiki?

Reply:
> > ![image](https://user-images.githubusercontent.com/29047033/145937756-66a87d9a-7efa-49fb-b1a1-86373a5aba88.png)
> > This is what it looks with `text-transform: uppercase` and `letter-spacing: .1em`, still kind of hard to read but acceptable imho.
> 
> Yeah, doesn't look so great. Should we check with the design team for suggestions?

I think we should, body-2 is too large for a error and there's nothing hierarchically below

Reply:
This is more of a TODO than a note. While this should probably be a JIRA ticket that we should iterate on, putting this in the wiki won't help anyone if we keep glossing over it when refactoring/reimplementing/augmenting an existing component.

In my experience that always seems to happen.